plugins/workflow/canceller.go

```diff
@@ -21,7 +20,7 @@ func (c *canceller) discard(id uint64) {
 	c.ids.Delete(id)
 }
 
-func (c *canceller) cancel(env bindings.WorkflowEnvironment, ids ...uint64) error {
+func (c *canceller) cancel(ids ...uint64) error {
 	var err error
 	for _, id := range ids {
 		cancel, ok := c.ids.LoadAndDelete(id)
```

Comment:
`LoadandDelete` is only `go.1.15` feature. It won't work in 1.14 and earlier. We should `Load` and then `Delete` until the minimum supported version is 1.15,